api/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/ceph/ceph-csi-operator/api
+
+go 1.22.5
```

Comment:
can you pleae use https://github.com/ceph/ceph-csi-operator/blob/4919ece1961894cca23a1bf13d80aff1bac58304/go.mod#L3-L4 same here